nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -194,7 +197,7 @@ let
             ${concatMapStringsSep "\n" listenString redirectListen}
 
             server_name ${vhost.serverName} ${concatStringsSep " " vhost.serverAliases};
-            ${optionalString vhost.enableACME acmeLocation}
+            ${acmeLocation}
```

Comment:
While it probably doesn't hurt having that there shouldn't it depend on `(vhost.enableACME || vhost.useACMEHost != null)` like the other parts?

Reply:
I do not understand what you mean. Unless `(vhost.enableACME || vhost.useACMEHost != null)` is true, the variable will contain an empty string. I chose to move the condition back to `acmeLocation` definition in order to reduce duplication.

Reply:
Yes, you are right. I see that now.